xilem/src/view/textbox.rs

```diff
@@ -73,7 +81,8 @@ impl<State: 'static, Action: 'static> View<State, Action, ViewCtx> for Textbox<S
         // TODO: Maybe we want a shared TextArea View?
         let text_area = widgets::TextArea::new_editable(&self.contents)
             .with_brush(self.text_brush.clone())
-            .with_alignment(self.alignment);
+            .with_alignment(self.alignment)
+            .with_insert_newline(self.insert_newline);
```

Comment:
This needs to be handled in rebuild as well; that is, in `rebuild`, we should call `set_insert_newline`.

That does raise questions about how to handle text which already has a newline in it (and `InsertNewline::OnEnter`)

Reply:
I've wired up the `rebuild`. When you say "text with newlines already in it" do you mean if someone changes the `InsertNewLine` behavior and then re-renders? It seems to me that in that case we should do nothing, the behavior - whether creating a newline or firing a TextEntered action - seems to me like it should be triggered by the enter key, not by the presence of newlines in the text?

Reply:
Ah I see. That feels like it's a slightly different kind of configuration to do with what characters are allowed in the textbox? The same problem would arise when pasting text for example, or when trying to enter non-numeric characters into a numeric field of some kind.

Reply:
Indeed. When we've had this discussion before, pasting has been the reason. Adding this into the mix just makes things even more complicated!